lua/neo-tree/setup/deprecations.lua

```diff
@@ -118,8 +118,8 @@ See instructions in `:h neo-tree-events` for more details.
 event_handlers = {
   {
     event = "neo_tree_popup_input_ready",
-    ---@param input NuiInput
-    handler = function(input)
+    ---@param args { bufnr: integer, winid: integer }
+    handler = function(args)
```

Comment:
The example here should perfectly replicate the old behavior, which mans it needs an `<esc>` mapping, right?

Reply:
Done.